test-plan.md

```diff
@@ -0,0 +1,100 @@
+# Validating C# Extension for VS Code
+
+#### Opening projects
+When you open a directory in VS Code, the C# extension should look for a .csproj or .sln file in that directory and use "OmniSharp" to load it. 
+If you look in "Output > Omnisharp Log" a bunch of information should be printed about what copy of MSBuild was used and what projects were load
+
+Project types to test:
+* Standalone csproj 
```

Comment:
Where do we expect these projects to come from?

Reply:
I took a workitem for myself to create this corpus: https://github.com/OmniSharp/omnisharp-vscode/issues/2079